appengine/flexible/tasks/create_app_engine_queue_task.py

```diff
@@ -31,16 +29,16 @@ def seconds_from_now_to_rfc3339_datetime(seconds):
 def create_task(project, queue, location, payload=None, in_seconds=None):
     """Create a task for a given queue with an arbitrary payload."""
 
+    from googleapiclient import discovery
```

Comment:
Prefer `import googleapiclient.discovery` and `client = googleapiclient.discovery.build(...)` throughout.

Reply:
Done